code/datums/abilities/wraith.dm

```diff
@@ -193,14 +193,14 @@
 			if (amt >= wraithMob.formaldehyde_tol)
 				interrupt(INTERRUPT_ALWAYS)
 				M.reagents.remove_reagent("formaldehyde", amt)
-				boutput(wraithMob, "<span class='alert'>This vessel is tainted with an... unpleasant substance... It is now removed...</span>")
+				wraithMob.visible_message("<span class='alert'>This vessel is tainted with an... unpleasant substance... It is now removed...</span>")
```

Comment:
correct yeah this is a bad idea

Reply:
so should i change this to boutput or is it fine as it is because thinking about it ghosts will see this message too

Reply:
it would've been kept the same, a boutput

Reply:
it is wrong to make it a visible message because it's written from the wraith's perspective and would make no sense to see as someone else